reference/enterprise-api-ref/inventory.markdown

```diff
@@ -73,7 +73,7 @@ curl -k --user <username>:<password> \
         "sort":"Host name",
         "filter":{
            "Host name":{
-              "matches":"value1",
+              "matches":["value1","value"],
```

Comment:
What is the logic for applying multiple filters? OR or AND?

Can you add that to the doc here?

Reply:
@nickanderson done and squashed